tests/postgres/grammar/test_select_grammar.py

```diff
@@ -303,26 +303,26 @@ def can_compile_join_clause(self):
         """
         return """SELECT * FROM "users" INNER JOIN "report_groups" AS "rg" ON "bgt"."fund" = "rg"."fund" AND "bgt"."dept" = "rg"."dept" AND "bgt"."acct" = "rg"."acct" AND "bgt"."sub" = "rg"."sub\""""
 
-    def can_compile_join_clause_with_where(self):
+    def can_compile_join_clause_with_value(self):
         """
         builder = self.get_builder()
         builder.where("age", "not like", "%name%").to_sql()
```

Comment:
Can we just get these doctoring updated? They should just be the code we are using to generate the query.

Looks like they were wrong when we originally wrote the test but since we're in here already now it might be good to update these now

Reply:
Aaaah yes I completely overlooked that ;)
Updated now